ruby/CMakeLists.txt

```diff
@@ -138,7 +138,7 @@ if (result EQUAL 0)  # Have minitest
   endmacro()
 
   pn_add_test(
-    UNWRAPPED
+    INTERPRETED
```

Comment:
I made a mistake there, before. Does not really matter, only when one starts enabling asan.